### ISSUE TITLE
Mention adb start for launching app

### ISSUE THREAD

**Issue by NoahBres** [gamemanual0/gm0#111]
**What information should be added?**
Android Studio seems to not automatically launch the app a lot of the time (a bug in the newer versions it seems?) so when you press play and upload the app, the app doesn't actually start. So you have to go to your phone and manually open it up. The workaround is running this command through ADB:

```bash
adb shell am start -n com.qualcomm.ftcrobotcontroller/org.firstinspires.ftc.robotcontroller.internal.PermissionValidatorWrapper
```

This way, you don't need to touch your phone.

**Does it need it's own page? (Yes/No):** No

**If it needs a page, which section should it be in? If it doesn't, which page should it be under?**
Idk wherever the ADB stuff is.

**Comment by NoahAndrews** [comment]
Won't that monkey command send a random input to the app? That feels like a bad way to launch it. The standard way should work fine, I'm surprised you said it didn't.

**Comment by NoahBres** [comment]
Well, to add onto the many things I have been wrong about in the past few days, adb shell start does in fact work. I am quite sure it did not work at one point? I documented which commands I tried in the Discord and asked about when originally trying it out last year. I guess I missed something. Didn't have anyone to correct me :P

Specifying 1 as the last argument for that command will default to the launch app intent so it should be fine. Although probably not ideal :P
